modules/collapse/src/collapse/CollapseSimulator.bak

```diff
@@ -0,0 +1,864 @@
+package collapse;
```

Comment:
I think we can remove the .bak files, we'll see the modified files in the pull request and we have the older version in previous commits so no need to have the .bak files in this branch

Reply:
I agree. The .bak files were capt to more easily identify the files that had been changed. However, once we conclude this new version is working correctly, we can eliminate the .bak files.